.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*   @bzspi @cvauvarin @jmhautbois-claranet @pdecat @shr3ps @tchernomax
+*   @bzspi @cvauvarin @jmhautbois-claranet @pdecat @shr3ps @tchernomax @haedri
```

Comment:
Please keep the list sorted.

Reply:
How about referencing teams here instead?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
@pdecat yes, good idea. We have to solve some issues now, but we'll update this file aftewards.